seqr/views/utils/permissions_utils.py

```diff
@@ -53,10 +73,33 @@ def check_multi_project_permissions(obj, user):
     raise PermissionDenied("{user} does not have view permissions for {object}".format(user=user, object=obj))
 
 
+def _get_workspaces_user_can_view(user):
+    requested_fields = 'public,workspace.name,workspace.namespace,workspace.workspaceId'
+    workspace_list = list_anvil_workspaces(user, requested_fields)
+    workspaces = []
+    for ws in workspace_list:
+        workspace_name = '/'.join([ws['workspace']['namespace'], ws['workspace']['name']])
+        if not ws['public']:
+            if user.is_staff:
+                workspaces.append(workspace_name)
```

Comment:
I thought the rule for AnVIL projects is different.

Reply:
I'm not entirely sure how to respond to that. I outlined above what the behavior is supposed to be. If you can explain why you think it should be different I'm happy to talk about that, but in short no its crucial that we able to disable staff access for anvil projects

Reply:
Maybe `Q(workspace_name__isnull=True` shouldn't be taken out because when the project has a workspace configured it means the local permissions must be ignored. This is the same for both the staff users and normal users. But it is not user-friendly for the transitioning phase because it could cause some users who have never signed in with AnVIL may lose access to some projects.

Reply:
yeah, its the transition phase that I'm worried about here. I think it would be best for us if we could add the workspace to projects without worrying about people losing access. I do see your point that in the long term it would be better to ignore local permissions on projects with workspaces, but I think while we are transitioning we shouldn't do that